README.md

```diff
@@ -1 +1,92 @@
-# SOURCE CONTROL HANDS ON WORKSHOP
\ No newline at end of file
+# SOURCE CONTROL HANDS ON WORKSHOP
+
+## Table of content
+
+* [Slide](#Slide)
+* [Overall Git's structure](#Overall-Git's-structure)
+* [Remote VS Local](#Remote-VS-Local)
+* [Untracked files](#Untracked-Files)
+* [Tracked Files](#Tracked-Files)
+
+## Slide
+https://docs.google.com/presentation/d/1cCGnm1pLVwWVn6EGceNRsQSW8wvKztiTQKo3kP4yGHU/edit?usp=sharing
+
+### Key words from the slide
+* Source Control
+* Git
+* GitHub
+* Fork
+* Clone
+
+Some of the key words that you have gone through from the slide above might not yet been explained. But at the end of the workshop, I can promise, it will be crystal clear to you.
+
+## Overall Git's structure
+We now know that the practice of using source control is very beneficial to the project. However, it would not be a clever idea if you have to create your own version control system. And this is where Git shines.
+
+### Remote VS Local
```

Comment:
I think we should move this section (Remote VS Local) to the end. Then, students should be about to grasp the concept of history easier when we refer to the commit they just made.